app/views/not_found.scala.html

```diff
@@ -35,7 +35,7 @@
             location = appConfig.customsFinancialsFrontendHomepage,
             linkClass = "govuk-link govuk-link--no-visited-state",
             preLinkMessage = Some("cf.error.not-found.message.go-to-home-page-text1"),
-            postLinkMessage = Some("cf.error.not-found.message.go-to-home-page-text2")
```

Comment:
is the message used anymore?

Reply:
no it wasnt the next commit removes the message